package.json

```diff
@@ -27,5 +27,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11"
```

Comment:
Sorry I should have been more descriptive, this should be added after line 9, right above `scripts`